avm/res/app/managed-environment/main.bicep

```diff
@@ -73,32 +73,41 @@ var builtInRoleNames = {
   Contributor: subscriptionResourceId('Microsoft.Authorization/roleDefinitions', 'b24988ac-6180-42a0-ab88-20f7382dd24c')
   Owner: subscriptionResourceId('Microsoft.Authorization/roleDefinitions', '8e3af657-a8ff-443c-a75c-2fe8c4bcb635')
   Reader: subscriptionResourceId('Microsoft.Authorization/roleDefinitions', 'acdd72a7-3385-48ef-bd42-f606fba81ae7')
-  'Role Based Access Control Administrator (Preview)': subscriptionResourceId('Microsoft.Authorization/roleDefinitions', 'f58310d9-a9f6-439a-9e8d-f62e7b41a168')
-  'User Access Administrator': subscriptionResourceId('Microsoft.Authorization/roleDefinitions', '18d7d88d-d35e-4fb5-a5c3-7773c20a72d9')
+  'Role Based Access Control Administrator (Preview)': subscriptionResourceId(
```

Comment:
This and other formatting changes should be taken care by the related open PRs

Reply:
... as those PRs  should also already apply the formatting (since we removed the V1 enforcement) I don't expect any conflicts if we were to merge this one in first. 💪 
So, is the idea of the comment to block this one regardless until the x other PRs are merged ? Or just to undo the reformatting (which is automatically applied), so I'd need to add the new feature via Notepad.

Reply:
I'm a bit lost. Didn't we remove v2 formatting from the bulk PR for any module having a PR already open? What is the difference if we add the formatting in this PR?

Reply:
Nah, enforcing V1 one doesn't unformat 😄 . I need to remove the changes by setting the files back to their state in main, then enforce V1, apply the change anew, un-enforce V1 and hope for the best 😄

Reply:
The difference to the previous PR is that we now have removed the V1 formatter enforcement. So 'IF' any of the open PRs stay up to date with upstream, they should all apply the reformatting on their end automatically (if I'm not mistaken).

Reply:
Regardless - you answered my question. I'll try to look into undoing the changes later this week as it's too much manual work for the time of day 😄

Reply:
Nono absolutely. 😄 just talking about this PR.
If their PRs are merged before this one I may anyways run into a merge conflict. I don't think there is necessarily a good solution unless both use the same formatting.

Reply:
Resolving this as not relevant anymore.